jetbrains-core/src/software/aws/toolkits/jetbrains/ui/feedback/FeedbackDialog.kt

```diff
@@ -168,11 +189,19 @@ class FeedbackDialog(val project: Project, initialSentiment: Sentiment = Sentime
 
     init {
         super.init()
-        title = message("feedback.title")
+
+        title = if (!isCodeWhisperer) {
```

Comment:
nit: weird that `feedback.comment.textbox.title` is parameterized, but not `feedback.title`

Reply:
yes, this makes sense. Will address it in the follow up PR.